cdap-docs/reference-manual/source/http-restful-api/metadata.rst

```diff
@@ -192,17 +204,15 @@ in the response body (pretty-printed)::
    * - Status Codes
      - Description
    * - ``200 OK``
-     - The properties requested were returned as a JSON string in the body of the response
-   * - ``404 NOT FOUND``
-     - The entity or program for which properties are being retrieved was not found
+     - The properties requested were returned as a JSON string in the body of the response which can be empty if there are no properties associated with the entity or entity does not exist
```

Comment:
if there are no properties associated with the entity, or the entity does not exist

Reply:
Done.